kubernetes/windows/main.ps1

```diff
@@ -852,24 +867,24 @@ function Bootstrap-CACertificates {
 }
 
 function IsGenevaMode() {
-    $isGenevaLogsIntegration=$false
-    $isGenevaLogsMultitenancy=$false
+    $isGenevaLogsIntegration = $false
+    $isGenevaLogsMultitenancy = $false
     $genevaLogsIntegration = [System.Environment]::GetEnvironmentVariable("GENEVA_LOGS_INTEGRATION")
     $genevaLogsMultitenancy = [System.Environment]::GetEnvironmentVariable("GENEVA_LOGS_MULTI_TENANCY")
     $genevaLogsInfraNameSpaces = [System.Environment]::GetEnvironmentVariable("GENEVA_LOGS_INFRA_NAMESPACES")
```

Comment:
Can we remove all these unnecessary changes? It is becoming confusing to understand what exactly are the changes needed for this feature?